esp8266/Makefile

```diff
@@ -148,7 +148,7 @@ DRIVERS_SRC_C = $(addprefix drivers/,\
 SRC_S = \
 	gchelper.s \
 
-FROZEN_MPY_PY_FILES := $(shell find -L $(FROZEN_MPY_DIR) -type f -name '*.py')
+FROZEN_MPY_PY_FILES := $(shell find -L $(FROZEN_MPY_DIR) -type f -name '*.py') modules/upip.py modules/upip_utarfile.py
```

Comment:
I'm not sure I exactly understand. Note that with unix port, where there's "automation", this automation makes using user's frozen modules complicated (almost impossible?). So, please answer which choice I should make:

1. Use this patch
2. Just add files modules/upip.py, modules/upip_tarfile.py to the repo.

Note that if choice 2 is selected, I'll likely follow up with a similar change for unix port in some time. (And the only reason I submit this patch is because unix port already does similar thing.)